mlflow/server/js/src/components/App.js

```diff
@@ -23,7 +23,7 @@ class App extends Component {
       <Router>
         <div>
           <ErrorModal/>
-          <header className="App-header">
+          {process.env.HIDE_HEADER === 'true' ? undefined : <header className="App-header">
```

Comment:
Isn't it more common to write null instead of undefined for stuff like this in React?